src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1236,7 +1241,144 @@ subroutine eigencalving(domain, err)
+      real (kind=RKIND), dimension(:,:), pointer :: flowParamA, &
+                                        temperature, layerThickness
+      real (kind=RKIND), pointer :: config_default_flowParamA
+      integer, pointer :: nCells
+      integer, dimension(:), pointer :: cellMask
+      real (kind=RKIND), dimension(:), pointer :: vonMisesStress       
+      err = 0
+      
+      call mpas_pool_get_config(liConfigs, 'config_grounded_von_Mises_threshold_stress', config_grounded_von_Mises_threshold_stress)
+      call mpas_pool_get_config(liConfigs, 'config_floating_von_Mises_threshold_stress', config_floating_von_Mises_threshold_stress)
+      !call mpas_pool_get_config(liConfigs, 'config_default_flowParamA',
+      !config_default_flowParamA) ! REMOVE THIS ONCE YOU CAN GET A FROM
+      !ALBANY!!!!!
+      call mpas_pool_get_config(liConfigs, 'config_flowLawExponent', config_flowLawExponent)
+      call mpas_pool_get_config(liConfigs, 'config_use_Albany_flowA_eqn', config_use_Albany_flowA_eqn)
+
+      block => domain % blocklist
+
+      do while (associated(block))
+         ! get pools
+          call mpas_pool_get_subpool(block % structs, 'geometry', geometryPool)
+          call mpas_pool_get_subpool(block % structs, 'velocity', velocityPool)
+          call mpas_pool_get_subpool(block % structs, 'mesh', meshPool)
+          call mpas_pool_get_subpool(block % structs, 'scratch', scratchPool)
+          call mpas_pool_get_subpool(block % structs, 'thermal', thermalPool)
+         ! get fields
+          call mpas_pool_get_dimension(meshPool, 'nCells', nCells)
+          call mpas_pool_get_array(geometryPool, 'cellMask', cellMask)
+          call mpas_pool_get_array(geometryPool, 'layerThickness', layerThickness)
+          call mpas_pool_get_array(velocityPool, 'eMax', eMax)
+          call mpas_pool_get_array(velocityPool, 'eMin', eMin)
+          call mpas_pool_get_array(velocityPool, 'vonMisesStress', vonMisesStress)
+          call mpas_pool_get_array(velocityPool, 'flowParamA', flowParamA)
+          call mpas_pool_get_array(velocityPool, 'xvelmean', xvelmean)
+          call mpas_pool_get_array(velocityPool, 'yvelmean', yvelmean)
+          call mpas_pool_get_array(geometryPool, 'calvingThickness', calvingThickness)
+          call mpas_pool_get_array(geometryPool, 'calvingVelocity', calvingVelocity) 
+          call mpas_pool_get_array(geometryPool, 'thickness', thickness)
+          call mpas_pool_get_array(thermalPool, 'temperature', temperature)
+          
+          vonMisesStress(:) = 0.0_RKIND
+
+          ! get flowParamA from MPAS or use Albany-like equation
+          if ( config_use_Albany_flowA_eqn ) then
+          !calculate Albany-type flowParamA
+          else
+             call li_calculate_flowParamA(meshPool, temperature, thickness,flowParamA,err) ! Get MPAS flowParamA
+          endif
+    
+         !Using a depth-averaged ice viscosity parameter B_depthAvg
```

Comment:
Does this bit of code need to remain for any reason?

Reply:
This was always a comment, just to make it easier to understand when comparing the von Mises calculation here to the equations in publications that use B. I'd prefer to leave it, but I can rephrase it to be more clearly just an explanatory note if you would like.

Reply:
My confusion is that what's commented is identical to what's in the vonMisesStress calculation a few lines down.  So if that's the version you think is better to use, I don't see a reason to keep the commented out version.

Reply:
Right, the reason I had that was to show that we are replacing the term B used in publications with a calculation using our available variables. It is not identical to the entire vonMisesStress calculation, just one term in it.